airbyte-integrations/connectors/source-bigquery/metadata.yaml

```diff
@@ -21,7 +21,7 @@ data:
             type: GSM
   connectorType: source
   definitionId: bfd1ddf8-ae8a-4620-b1d7-55597d2ba08c
-  dockerImageTag: 0.4.4
+  dockerImageTag: 0.4.5
```

Comment:
Minor version change suggests change is not breaking. 
Is this correct?
I'm asking because of update to spec json . Will existing connection continue to work after upgrading ?